app/consumer-democracy/app.go

```diff
@@ -82,30 +82,31 @@ import (
 	dbm "github.com/tendermint/tm-db"
 
 	distr "github.com/cosmos/cosmos-sdk/x/distribution"
-	ccvdistrclient "github.com/cosmos/cosmos-sdk/x/distribution/client"
-	ccvdistrkeeper "github.com/cosmos/cosmos-sdk/x/distribution/keeper"
-	ccvdistrtypes "github.com/cosmos/cosmos-sdk/x/distribution/types"
+	distrclient "github.com/cosmos/cosmos-sdk/x/distribution/client"
+	distrkeeper "github.com/cosmos/cosmos-sdk/x/distribution/keeper"
+	distrtypes "github.com/cosmos/cosmos-sdk/x/distribution/types"
 	"github.com/cosmos/interchain-security/testutil/e2e"
 	ccvdistr "github.com/cosmos/interchain-security/x/ccv/democracy/distribution"
 
-	ccvstakingkeeper "github.com/cosmos/cosmos-sdk/x/staking/keeper"
-	ccvstakingtypes "github.com/cosmos/cosmos-sdk/x/staking/types"
+	stakingkeeper "github.com/cosmos/cosmos-sdk/x/staking/keeper"
+	stakingtypes "github.com/cosmos/cosmos-sdk/x/staking/types"
 	ccvstaking "github.com/cosmos/interchain-security/x/ccv/democracy/staking"
 
 	gov "github.com/cosmos/cosmos-sdk/x/gov"
-	ccvgovkeeper "github.com/cosmos/cosmos-sdk/x/gov/keeper"
-	ccvgovtypes "github.com/cosmos/cosmos-sdk/x/gov/types"
+	govkeeper "github.com/cosmos/cosmos-sdk/x/gov/keeper"
+	govtypes "github.com/cosmos/cosmos-sdk/x/gov/types"
 	ccvgov "github.com/cosmos/interchain-security/x/ccv/democracy/governance"
 
 	// add mint
-	ccvmint "github.com/cosmos/cosmos-sdk/x/mint"
-	ccvmintkeeper "github.com/cosmos/cosmos-sdk/x/mint/keeper"
-	ccvminttypes "github.com/cosmos/cosmos-sdk/x/mint/types"
+	mint "github.com/cosmos/cosmos-sdk/x/mint"
+	mintkeeper "github.com/cosmos/cosmos-sdk/x/mint/keeper"
+	minttypes "github.com/cosmos/cosmos-sdk/x/mint/types"
 
 	paramproposal "github.com/cosmos/cosmos-sdk/x/params/types/proposal"
-	ibcconsumer "github.com/cosmos/interchain-security/x/ccv/consumer"
+	consumer "github.com/cosmos/interchain-security/x/ccv/consumer"
+	consumerkeeper "github.com/cosmos/interchain-security/x/ccv/consumer/keeper"
```

Comment:
```suggestion
             consumerkeeper "github.com/cosmos/interchain-security/x/ccv/consumer/keeper"
	ibcconsumerkeeper "github.com/cosmos/interchain-security/x/ccv/consumer/keeper"
```

These 2 are the same, possibly `ibcconsumerkeeper` is used in unchanged parts of the code which so they are not displayed in the diff

Reply:
Nice catch 👍, see https://github.com/cosmos/interchain-security/pull/679/commits/84794a9fe1f83b250947e57e03c72c8402a2b3f0